src/ce/ti/getcsc.h

```diff
@@ -93,19 +93,26 @@ sk_key_t os_GetCSC(void);
 #define sk_6                0x13
 #define sk_9                0x14
 #define sk_Comma            0x25
-#define sk_Sin              0x26
-#define sk_Apps             0x27
-#define sk_GraphVar         0x28
+#define sk_Sin              0x26 /**< 84+CE.  On the TI-83 Premium CE, this is [TRIG]  */
+#define sk_Apps             0x27 /**< 84+CE.  On the TI-83 Premium CE, this is [MATRICE] */
+#define sk_GraphVar         0x28 /**< X, T, theta, n */
 #define sk_DecPnt           0x19
 #define sk_LParen           0x1D
-#define sk_Cos              0x1E
+#define sk_Cos              0x1E /**< 84+CE.  On the TI-83 Premium CE, this is [RESOL]. */
 #define sk_Prgm             0x1F
 #define sk_Stat             0x20
-#define sk_Chs              0x11
+#define sk_Chs              0x11 /**< Unary negation, not subtraction */
 #define sk_RParen           0x15
-#define sk_Tan              0x16
+#define sk_Tan              0x16 /**< 84+CE.  On the TI-83 Premium CE, this is the fraction template. */
 #define sk_Vars             0x17
-#define sk_Power            0x0E
+#define sk_Power            0x0E /**< [^] */
+
+/* For the TI-83 Premium CE */
+#define sk_AnsFrac          sk_Recip /**< 83PremCE: [<>].  I have no idea what TI actually calls this.  It's [x^-1] on the TI-84 Plus CE. */
```

Comment:
Oh so you did put them here sorry.
Unless that existed before and must be reused for compatibility this isn't actually just ans<->frac but an approx<->exact toggle, the problem is that I'm not sure what a better name would be... toggleAns?

Reply:
I'm renaming `sk_AnsFrac` to `sk_ExactTog` because `sk_ToggleAns` doesn't tell you *how* Ans is being toggled.